cocos/2d/CCCameraBackgroundBrush.cpp

```diff
@@ -279,10 +279,26 @@ bool CameraBackgroundColorBrush::init()
     return true;
 }
 
+void CameraBackgroundColorBrush::drawBackground(Camera* camera)
+{
+    GL::blendFunc(_blendFunc.src, _blendFunc.dst);
+
+    CameraBackgroundDepthBrush::drawBackground(camera);
+}
+
 void CameraBackgroundColorBrush::setColor(const Color4F& color)
 {
     _quad.bl.colors = _quad.br.colors = _quad.tl.colors = _quad.tr.colors = Color4B(color);
     
+    if (color.a != 1.f)
+    {
+        _blendFunc = BlendFunc::ALPHA_NON_PREMULTIPLIED;
+    }
+    else
+    {
+        _blendFunc = BlendFunc::ALPHA_PREMULTIPLIED;
+    }
```

Comment:
When alpha = 1.f, then `BlendFunc::ALPHA_PREMULTIPLIED` == `BlendFunc::ALPHA_NON_PREMULTIPLIED`.  So i think there is not need to set blend function here.

Reply:
just initialize it in constructor.